docs/getting-started/generated-app.md

```diff
@@ -0,0 +1,53 @@
+---
+id: generated-app
+title: The generated app
+sidebar_label: The Generated App
+slug: /generated-app
+---
+
+# The Generated App
+
+Applications created with Amplication are generated as human-readable and editable TypeScript code.
+
+In this article you will find the details on the structure of the application and the technologies behind it.
+
+## Technologies
+
+The generated app is built with the following amazing open-source technologies:
```

Comment:
@yuval-hazaz can you please remove 'amazing'? thanks

Reply:
How about "proven"?